lib/restrictor_mma.in.cu

```diff
@@ -0,0 +1,279 @@
+#include <color_spinor_field.h>
+#include <multigrid.h>
+#include <power_of_two_array.h>
+#include <tunable_block_reduction.h>
+#include <expand_list.hpp>
+
+#ifdef QUDA_MMA_AVAILABLE
+#include <device.hpp>
+#include <kernels/restrictor_mma.cuh>
+#include <mma_tensor_op/smma_m16n8k8_sm70.cuh>
+#endif
+
+namespace quda
+{
+
+#ifdef QUDA_MMA_AVAILABLE
+
+  template <typename out_t, typename in_t, typename v_t, int fineSpin, int fineColor, int coarseSpin, int coarseColor, int nVec>
+  class RestrictMmaLaunch : public TunableKernel
+  {
+    ColorSpinorField &out;
+    const ColorSpinorField &in;
+    const ColorSpinorField &v;
+    const int *fine_to_coarse;
+    const int *coarse_to_fine;
+    const int parity;
+    int aggregate_size;
+
+    using mma_t = typename mma::mg_mma_restrictor_t<out_t>::type;
+
+    static constexpr int spin_block_factor = spin_mapper<fineSpin, coarseSpin>::get_spin_block_factor();
```

Comment:
some comments about the numerical values of `aggregate_size_block_max`, `block_atom_size` and `block_limit` would be helpful

Reply:
https://github.com/lattice/quda/pull/1497/commits/87d332f5178d97dc15e642d362f1e5d4e1f2bfd8